api/tests/routes/backoffice_v3/tags_test.py

```diff
@@ -91,6 +109,9 @@ def test_update_tag(self, authenticated_client):
         assert tag.description == new_tag_description
         assert tag.startDateTime.date() == new_start_date
         assert tag.endDateTime.date() == new_end_date
+        assert len(tag.categories) == 2
+        assert tag.categories[0].label == "Comptage partenaire EPN"
+        assert tag.categories[1].label == "Playlist lieux et offres"
```

Comment:
(détail) Je ne pense pas qu'on puisse garantir l'ordre de sauvegarde ni l'ordre de récupération des catégories. Pour éviter un éventuel test échouant aléatoirement, on pourrait tester ainsi : 

```suggestion
        assert {cat.label for cat in tag.categories} == {"Comptage partenaire EPN", "Playlist lieux et offres"}
```